### PR TITLE
fix(wizard): use green color for complete steps

### PR DESCRIPTION
**Pull request by rosendi**
Use the green color in wizards for complete steps.

### CODE REVIEW

File: src/blocks/wizard/__step/__step.css [hunk 1]
```diff
@@ -175,7 +175,7 @@
   display: inline-flex;
 }
 .wizard__step_state_done:not(.wizard__step_active) .wizard__stepTitleIndexCheck svg {
-  fill: #1E88E5;
+  fill: #4caf50;
```
Comment (radiansz) [inline src/blocks/wizard/__step/__step.css]: Did you change in only in css file?

Comment (rosendi) [inline src/blocks/wizard/__step/__step.css]: It's kill me all the time!

Comment (rosendi) [inline src/blocks/wizard/__step/__step.css]: @radiansz It's done!

**Comment by radiansz** [comment]
LGTM

**Comment by trikashny** [comment]
LGTM

**Pull request merged**